### PR TITLE
Araneus is now a real spider, and has invaded Boxstation.

### PR DESCRIPTION
**Pull request by Firecage**
As a side note, I even redid the whole thing for metastation, while the mapmerger correctly did it on box...for some reason for meta it keeps showing a fuckload of lines changed each time. Even the bloody mapmerger said 8k lined changed....even when I merely replaced the edited bat with a real spider Araneus.

Anyway. This turns Araneus into a real spider and puts said real spider into the HoS's office on Meta. This also gives the Box HoS an Araneus.

Edit: Mmm, also. Apparently won't be able to attack human enemies of the station, like OPs, since they still have the faction of neutral. The same as the station.

### CODE REVIEW

**Comment by neersighted** [comment]
On meta you could just edit the map by hand

**Comment by Firecage** [comment]
Neersighted, I tried both methods with seperate map files. First by editing the map directly, second by using notepad++ to just change the mob manually.....both times, mapmerger did this.

**Comment by neersighted** [comment]
Just edit meta without using the map merger?

**Comment by Firecage** [comment]
Nah, Neersighted, then it will be even worse. The mapmerger is exactly to reduce the amount of lines changed(...and to make merge conflicts less likely...). Pretty sure it will be even worse without the map merger

**Comment by MrStonedOne** [comment]
firecage, when editing with notepad++ you aren't suppose to use the map merger.

map merger is just for undoing the things byond does to map files that cause so many changed lines

**Comment by Firecage** [comment]
Fuck. Really MSO?....and the entire time with map edits with N++ I was using the map merger...

Regardless, with both methods still 8k line changes for some reason.

**Comment by MrStonedOne** [comment]
[06:29:19] \<Firecage\> Well, regardless, no matter what method I did, Metastation still caused 8k line changes
[06:29:22] \<Firecage\> merely for replacing Araneus
[06:29:43] <+MrStonedOne> oh
[06:29:44] <+MrStonedOne> i know
[06:29:49] <+MrStonedOne> you can blame oranges for this one
[06:29:55] <+MrStonedOne> line endings me bets

**Comment by ChangelingRain** [comment]
This means the spider won't retaliate if you attack it, and that's terrible.

**Comment by Firecage** [comment]
ChangelingRain. Perhaps, but it's better than having fake spiders. Aka resprited bats. Just give the HoS a bat pet if you want it then.

File: code/modules/mob/living/simple_animal/hostile/giant_spider.dm [hunk 1]
```diff
@@ -69,6 +69,18 @@
 		return
 	key = user.key
 
+/mob/living/simple_animal/hostile/poison/giant_spider/araneus
+	name = "Sergeant Araneus"
+	desc = "A fierce companion for any person of power, this spider has been carefully trained by NanoTrasen specialists. Its beady, staring eyes send shivers down your spine."
```
Comment (neersighted) [inline code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: Nanotrasen

Comment (Firecage) [inline code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: Actually, I am pretty sure canon-wise, it is NanoTrasen.

Comment (Cheridan) [inline code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: you're wrong

Comment (MrStonedOne) [inline code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: This is being discussed in #lorebus

We will have the official results shortly.

Comment (MrStonedOne) [inline code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: http://pastebin.com/FeLGRsb2

**Comment by optimumtact** [comment]
What don't blame me for this

just edit the map manually, don't use the map merger

**Comment by optimumtact** [comment]
Okay yeah that probably was me

File: code/modules/mob/living/simple_animal/hostile/giant_spider.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: Official living lore indicates it's Nanotrasen

https://github.com/tgstation/-tg-station/search?utf8=%E2%9C%93&q=nanotrasen

**Comment by Lumipharon** [comment]
>araneus won't even attack people anymore

What is even the point?

**Comment by Firecage** [comment]
Eh. I'le later try to move all the /hostile/retaliate/ paths shit to just /hostile/, so any hostile mob can be set to retaliate instead of attack. But eeeeeh, fucked up last time with it. But for now, guess PR goes sleepy night.

**Pull request closed without merging**